src/Aspire.Hosting.Azure.AppService/AzureAppServiceWebsiteContext.cs

```diff
@@ -99,7 +99,7 @@ private void ProcessEndpoints()
                 Scheme: endpoint.UriScheme,
                 Host: HostName,
                 Port: endpoint.UriScheme == "https" ? 443 : 80,
-                TargetPort: null, // App Service manages internal port mapping
+                TargetPort: endpoint.TargetPort,
```

Comment:
Was the comment in correct?

Reply:
Not fully. It works if the docker file uses exposes but we want to use whatever is defined in the appmodel as the source of truth